pkg/bucket/bandwidth/monitor.go

```diff
@@ -0,0 +1,170 @@
+/*
+ * MinIO Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package bandwidth
+
+import (
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/minio/minio/pkg/bandwidth"
+	"github.com/minio/minio/pkg/bucket/bandwidth/measurements"
+	"github.com/minio/minio/pkg/pubsub"
+)
+
+// GetThrottler gets the throttler for bucket and sets the bandwidth to the configured value
+func (m *Monitor) GetThrottler(bucket string, bandwidthBiPS int64) *Throttler {
```

Comment:
Don't mix functionality. A 'get' shouldn't update the bandwidth.

Reply:
I will rename it, I want to avoid an explicit logic to update the bandwidth if the config changes.